mungegithub/mungers/approvers/owners.go

```diff
@@ -192,9 +190,26 @@ func removeSubdirs(dirList []string) sets.String {
 	return finalSet
 }
 
+// Approval has the information about each approval on a PR
+type Approval struct {
+	Name      string // Name of the approver
```

Comment:
Can we change this to Login to be consistent with githubapi convention (there's a Name and Login section and I think we use Login)